misc/python/materialize/data_ingest/rowlist.py

```diff
@@ -0,0 +1,23 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from typing import List
+
+from materialize.data_ingest.row import Row
+
+
+class RowList:
+    rows: List[Row]
+    # generator_properties
```

Comment:
This comment confuses me more than it helps. Is there something missing?

Reply:
That was a TODO for myself, made it clearer.